issues.md

```diff
@@ -0,0 +1,26 @@
+# Issue reporting guidelines
+
+## General guidelines
+
+An issue must be an actionable task.
```

Comment:
Not a blocker, but I am wondering about 'tracking issues' - these are the kind of main issues OWD use to outline lots of tasks needed for a project

Reply:
edited in https://github.com/mdn/temp-processes/pull/1/commits/e4c07c837c75b1145c7121fd5993b5c312f24ea7